src/pages/Travel/DomainSelectorPage.tsx

```diff
@@ -62,7 +64,10 @@ function DomainSelectorPage() {
                         success
                         large
                         style={[styles.w100]}
-                        onPress={() => provisionDomain(selectedDomain ?? CONST.TRAVEL.DEFAULT_DOMAIN)}
+                        onPress={() => {
```

Comment:
Please move this to it's own function outside the JSX.

Reply:
moved, but why do we do this? I saw this in our codebase, but it is unclear why. Is it to avoid creating a function every time the view renders? If so, I'm afraid this doesn't help since we need the `selectedDomain` variable from the view 🤔

Reply:
To be clear, I'm not asking to move it outside of the component, just outside of the `return (...);` part of the component. It should look more like:

```
const myComponent = () => {
    const provisionDomain = () => {
        // Method goes here
    };

    return (....);
};
```

The purpose is to separate the rendering logic from the UX logic so that it makes for a cleaner component that is easier to see what's going on, and more consistent with previous code patterns.